site/en/docs/privacy-sandbox/attribution-reporting/working-with-noise/index.md

```diff
@@ -135,8 +135,6 @@ An important part of allocating budget between different keys is understanding h
 
 Lower values of epsilon provide greater privacy protection.
 
-During the origin trial, ad tech companies can choose epsilon within a given range. We recommend you start with epsilon=10. Epsilon can be set to any value between 0 and 64: 0 < epsilon ≤ 64. This range is wide enough to allow for flexible testing.
```

Comment:
@maudnals (sorry to ask you more PS question!): Was this only possible during the origin trial, or can epsilon still be set?

Reply:
Akash confirmed users can stiil set epsilon and the range is the same